### PR TITLE
fix(content-group-pictograms): fixed bottom spacing

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#2624 

### Description

Updated spacing on `ConentGroupPictograms` bottom.

### CODE REVIEW

File: packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss [hunk 1]
```diff
@@ -15,6 +15,10 @@
       }
     }
 
+    .#{$prefix}--content-group {
+      margin-bottom: $layout-05;
+    }
+
```
Comment (annawen1) [inline packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss:21]: hey @RaphaelAmadeu I'm looking at the ticket, and @ltitle also mentioned about pictogram bottom padding needing to be removed for the very last pictogram item

Comment (RaphaelAmadeu) [reply packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss:21]: @annawen1 these remove the padding from the content-group itself, in the end both would have the same result.

Comment (annawen1) [reply packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss:21]: I guess that works...I'm a little concerned about other components placed into ContentGroup, will they always have 32px padding bottom?

Comment (annawen1) [reply packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss:21]: ah I was thinking generically, this is specifically the `_content-group-pictograms` styles so this works! thanks :D

**Comment by RaphaelAmadeu** [comment]
reopened

**Pull request merged**